_posts/2021-01-08-2020-review.md

```diff
@@ -0,0 +1,264 @@
+%}).
+
+Brian and Manfred launched the live streaming event [Trino Community
+Broadcast](broadcast/index.html), and grew their audience and back catalog to
+include some very useful material. If you have not seen it yet, go and [watch
+some old episodes](/broadcast/episodes.html) and join us in the next ones.
+
+We also had a number of other online events and presentations, with direct
+participation of our community members:
+
+* A [dedicated conference event](% post_url 2020-11-21-a-report-about-presto-conference-tokyo-2020 %})
+  for the community in Japan was very successful.
+* The [Argentina Big Data Meetup](% post_url
+  2020-09-28-argentina-big-data-meetup %}) had a large audience from the
+  community in South America
+
+A series of virtual events around the project started with a roadmap and
+overview meeting and included a number real world use case examples at scale:
+
+* [State of Trino]({% post_url 2020-05-15-state-of-presto %})
+* [Trino at Pinterest]({% post_url 2020-07-22-presto-summit-pinterest %})
+* [Trino Migration at ARM Treasure Data]({% post_url 2020-07-06-presto-summit-arm-td %})
+* [Trino at Zuora]({% post_url 2020-06-16-presto-summit-zuora %})
+
+Another series of training classes with the project founders was hugely
+successful. It includes very valuable content for any Trino user, from beginners
+to experts, that you should not miss:
+
+* [Advanced SQL in Trino with David]({% post_url 2020-07-15-training-advanced-sql %})
+* [Understanding and Tuning Trino Query Processing with Martin]({% post_url 2020-07-30-training-query-tuning %})
+* [Securing Trino with Dain]({% post_url 2020-08-13-training-security %})
+* [Configuring and Tuning Trino with Dain]({% post_url 2020-08-27-training-performance %})
+
+## Features and code
+
+While everything mentioned is already exciting, the true work is visible in the
+new features and improvements in Trino. It is a long list, but read on. You
+won’t want to miss anything.
+
+### Improvements to ANSI SQL support
+
+A core feature of Trino is the ability to use the same standard SQL for any
+connected data source. These improvements empower all users.
+
+* Variable-precision temporal types, with precision down to picoseconds
+  (10<sup>−12</sup>s). This a very important feature for any time critical
+  systems such as financial transactions processing
+* Correct, and now SQL specification compliant timestamp semantics, making
+  migration of SQL statements from other compliant systems such as many RDBMSs
+  easier
+* Implicit coercions for `INSERT` clause.
+* Support for `RANGE` and `GROUPS`-based window frames
+* More support for various shapes of correlated subqueries
+* Support for `INTERSECT ALL` and `EXCEPT ALL`
+* Experimental support for [recursive queries](/docs/current/sql/select.html?highlight=recursive#with-recursive-clause)
+* Enforcement of `NOT NULL` constraints when inserting data
+* Quantified comparisons (e.g., > ALL (...)) in aggregations
```

Comment:
@martint is parameter support in LIMIT / FETCH FIRST / OFFSET clause worth mentioning?
And maybe WINDOW clause and MATCH_RECOGNIZE as ongoing work?

Reply:
> is parameter support in LIMIT / FETCH FIRST / OFFSET clause worth mentioning?

I think it's fairly minor, but we could mention it.

> And maybe WINDOW clause and MATCH_RECOGNIZE as ongoing work?

Yes, those should be in the section at the end where we talk about upcoming projects.